express-api/src/services/properties/propertiesServices.ts

```diff
@@ -818,7 +818,6 @@ const getPropertiesUnion = async (filter: PropertyUnionFilter) => {
 
   // Add quickfilter part
   if (filter.quickFilter) {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Any idea why these are no longer being picked up as errors?
Perhaps this is not being defined in the configuration anymore.

Reply:
> Any idea why these are no longer being picked up as errors? Perhaps this is not being defined in the configuration anymore.

I have set the rule @typescript-eslint/no-explicit-any to 'warn' in eslint.config.js. This means ESLint will warn you whenever it detects the use of any, but it will not fail your build or force you to disable the rule with comments like eslint-disable-next-line.